src/server/api/routers/matchRequest.ts

```diff
@@ -299,8 +293,9 @@ export const matchRequestRouter = createTRPCRouter({
             matchType: true,
           },
         });
+      };
 
-      await mutextProtectedTryMatch(curUserId, matchedRequest);
+      await mutexProtectedTryMatch(curUserId, getMatchedRequest);
```

Comment:
This is redundant? We cannot have race when using findunique, cause it's a single prisma query

Reply:
Thanks for pointing it out, i think you're right. As discussed we'd just submit this to clear it since correctness should be there.